content/en/docs/components/pipelines/user-guides/components/containerized-python-components.md

```diff
@@ -62,17 +62,18 @@ src/
 In this step, you'll provide `base_image` and `target_image` arguments to the `@dsl.component` decorator of your component in `src/my_component.py`:
 
 ```python
-@dsl.component(base_image='python:3.7',
+@dsl.component(base_image='python:3.11',
```

Comment:
Can we use Python 3.9, which is still supported?

Reply:
We support any >= 3.9 and due to that it is the website. It is fine to use 3.11

Reply:
Shouldn't we use 3.12 then? I mean, the lowest or the highest.

Reply:
It can be anything as long as it matches one of the items declared in: https://github.com/kubeflow/pipelines/blob/master/sdk/python/setup.py#L100